docs/generated/sql/bnf/explain_analyze_stmt.bnf

```diff
@@ -1,3 +1 @@
 explain_stmt ::=
-	'EXPLAIN' 'ANALYZE' preparable_stmt
```

Comment:
This production is empty now. That's either an error or it should be removed completely.